doc/plan_stubs/external_code.md

```diff
@@ -0,0 +1,75 @@
+# `call_sync` (calling external code)
+
```

Comment:
I think we should link to the API ref here. 

```suggestion

API reference: {py:obj}`ibex_bluesky_core.plan_stubs.call_sync`

```

as per https://myst-parser.readthedocs.io/en/latest/syntax/code_and_apis.html#sphinx-autodoc2